Archiver.py

```diff
@@ -1,4 +1,30 @@
 
+import os.path
+
+
+def read_file(file_path):
+    file_path = os.path.abspath(file_path)
+    file_name = os.path.basename(file_path).split(".")[0]
+    try:
+        with open(file_path, "r") as input_file:
+            out_string = input_file.readline()
+    except FileNotFoundError:
+        print("So file is not found, try again")
```

Comment:
Such file not found
Requested file not found 
File not found
etc

Actually, it's a good idea to write your comments in english first and than try to translate them via some translator to ensure Your sentences were correct

Reply:
Agree fixed. I will  translate sentences into russian later before using them.